src/styles/block/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  padding: 10px;
+  width: 100px * 7 + 1px * 6;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  $weekdays:
+    mon 0,
+    tue 1,
+    wed 2,
+    thu 3,
+    fri 4,
+    sat 5,
+    sun 6;
```

Comment:
shouldn't be there

Reply:
I can`t connect my variables.scss file. Have an error in the cycle.